include/BSMPT/models/modeltests/ModelTestfunctions.h

```diff
@@ -63,5 +64,14 @@ TestResults CheckSymmetricTensorQuarks(
 TestResults CheckSymmetricTensorGauge(
     const std::vector<std::vector<std::vector<std::vector<double>>>> &Tensor);
 
+/**
+ * Checks if the tensors are correctly implemented. For this the fermion,
+ * quark and gauge boson masses are calculated and printed next to the values
+ * defined in SMparah.h
+ */
+void CheckImplementation(
+    const std::unique_ptr<Class_Potential_Origin> &modelPointer,
```

Comment:
```suggestion
    const Class_Potential_Origin &modelPointer,
```

We dont need it as a pointer in the function. The function calls all the subfunctions as a reference anyway

Reply:
Done